clients/banking/src/pages/AccountActivationPage.tsx

```diff
@@ -274,6 +276,7 @@ type Props = {
   canViewAccountDetails: boolean;
   projectName: string;
   refetchAccountAreaQuery: () => void;
+  hasPvidOrQesValidation: boolean;
```

Comment:
I hesitate to rename this prop `requireFirstTransaction`, because if the condition become more complexe in the futur, the name will not be valid anymore.  
What do you think?

Reply:
You're right, I changed the name with requireFirstTransfer instead